hazelcast/src/main/java/com/hazelcast/sql/impl/schema/map/PartitionedMapTable.java

```diff
@@ -39,4 +41,20 @@ public PartitionedMapTable(
     public PartitionedMapTable(String name, QueryException exception) {
         super(SCHEMA_NAME_PARTITIONED, name, exception);
     }
+
+    @Override
+    public PlanObjectId getObjectId() {
```

Comment:
"Object ID" sounds too generic to me. ID implies that it's stable, but it changes with any property change. Maybe "comparison key" would be better. Or "object key", for brevity. Also should contain "plan", because the key is tailored for plan. What about this?:
```suggestion
    public PlanObjectKey getPlanObjectKey() {
```

Reply:
Renamed